pallets/ddc-customers/Cargo.toml

```diff
@@ -46,6 +46,7 @@ std = [
   "scale-info/std",
   "sp-runtime/std",
   "sp-std/std",
+  "pallet-balances/std",
```

Comment:
balances in [dev-dependencies] and shouldn't probably require std featyre

Reply:
The tests fails without it